readme.md

```diff
@@ -17,11 +16,13 @@ following table shows the status of various features.
 | Document symbols      |   ✅   |                                                                                                  |
 | Formatting            |   ✅   | Uses Scalafmt.                                                                                   |
 | Find references       |   ✅   |                                                                                                  |
+| Find implementations  |   ✅   |                                                                                                  |
 | Workspace symbol      |   ✅   | Searches workspace sources and library dependencies. All-lowercase queries are case-insensitive. |
 | Highlight             |   ✅   |                                                                                                  |
 | Folding               |   ✅   |                                                                                                  |
-| Rename symbol         |   ❌   |                                                                                                  |
-| Code actions          |   ❌   |                                                                                                  |
+| Rename symbol         |   ✅   |                                                                                                  |
+| Code actions          |   ⚠️    | Limited code actions are currently available                                                     |
```

Comment:
I would remove this entry. "Code actions" is extremely generic and it's not very useful to have it here.